decode.go

```diff
@@ -514,6 +531,9 @@ type DecOptions struct {
 	// FieldNameByteString specifies the behavior when decoding a CBOR byte string map key as a
 	// Go struct field name.
 	FieldNameByteString FieldNameByteStringMode
+
+	// ReturnTypeForEmptyIface specifies the type of the value to be returned when decoding into an empty interface
+	ReturnTypeForEmptyIface ReturnTypeForEmptyInterface
```

Comment:
Maybe rename this.

```suggestion
	// UnrecognizedTagToAny specifies how to decode unrecognized CBOR tag into an empty interface.
	// Currently recognized CBOR tag numbers are 0, 1, 2, 3, or registered by TagSet.
	UnrecognizedTagToAny UnrecognizedTagToAnyMode
```

Reply:
Thanks for the suggestion. I've made these changes.